src/components/UserInfo/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Image } from "semantic-ui-react";
```

Comment:
This is an excellent use of a sub component, but consider putting it in the GameList folder. I'm not sure where else we would use it.